src/linker/Linker.Steps/OutputStep.cs

```diff
@@ -285,18 +285,26 @@ protected virtual void CopyAssembly (AssemblyDefinition assembly, string directo
 			if (source == target)
 				return;
 
-			CopyFileAndRemoveReadOnly (source, target);
+			File.Copy (source, target, true);
 
 			if (!Context.LinkSymbols)
 				return;
 
 			var mdb = source + ".mdb";
-			if (File.Exists (mdb))
-				CopyFileAndRemoveReadOnly (mdb, target + ".mdb");
+			if (File.Exists (mdb)) {
+				if (removeReadOnly)
+					CopyFileAndRemoveReadOnly (mdb, target + ".mdb");
```

Comment:
Why about changing 

`static void CopyFileAndRemoveReadOnly (string src, string dest)`

to

`static void CopyFile (string src, string dest, bool removeReadOnly)`